config-xml/addon/src/main/java/org/jboss/windup/config/parser/metadata/MetadataExecuteBeforeHandler.java

```diff
@@ -0,0 +1,20 @@
+package org.jboss.windup.config.parser.metadata;
+
+import org.jboss.windup.config.exception.ConfigurationException;
+import org.jboss.windup.config.parser.ElementHandler;
+import org.jboss.windup.config.parser.NamespaceElementHandler;
+import org.jboss.windup.config.parser.ParserContext;
+import org.w3c.dom.Element;
+
+@NamespaceElementHandler(elementName = "executeBefore", namespace = "http://windup.jboss.org/v1/xml")
+public class MetadataExecuteBeforeHandler implements ElementHandler<String>
+{
+
+    @Override
+    public String processElement(ParserContext handlerManager, Element element) throws ConfigurationException
+    {
+        String executeBefore = element.getTextContent();
```

Comment:
Need to throw a validation exception if this value is missing. (Like our existing handlers.)